packages/client/src/utils/buttonActions.js

```diff
@@ -21,6 +21,17 @@ const deleteRowHandler = async (action, context) => {
   })
 }
 
+const triggerAutomationHandler = async (action, context) => {
+  const params = {}
+  for (let field in action.parameters.fields) {
```

Comment:
I think you might be able use the `enrichDataBindings` helper here to parse all your values without looping.